api/xml/xml.go

```diff
@@ -495,7 +495,6 @@ func maritalStatus(status string) string {
 	alias := map[string]string{
 		"NeverMarried": "NeverMarried",
 		"Married":      "Married",
-		"InCivilUnion": "Married",
```

Comment:
Since we've removed InCivilUnion, this method can be removed – both where it is referenced in this file and in the places where it is referenced in the templates.

Reply:
This is because the eApp vernacular (left side) now matches the e-QIP vernacular (right side) in all the cases.

Reply:
Makes sense, removed the function and references to it.